### ISSUE TITLE
pip3 install version conflict for importlib-metadata

### ISSUE THREAD

**Issue by smallpize** [retspen/webvirtcloud#532]
When I downloaded the latest version for installation, run "pip3 install -r conf/requirements.txt -q" , pip installation prompted a version conflict for importlib-metadata.
log: 
ERROR: Cannot install -r conf/requirements.txt (line 22) and -r conf/requirements.txt (line 3) because these package versions have conflicting dependencies.
The conflict is caused by:
    django-icons 22.1 depends on importlib-metadata<3; python_version < "3.8"
    markdown 3.4.1 depends on importlib-metadata>=4.4; python_version < "3.10"

**Comment by catborise** [comment]
which OS are you using for webvirtcloud?

**Comment by smallpize** [comment]
> which OS are you using for webvirtcloud?

Kylin V10 SP2 (like openEuler-20.03-LTS-SP1)

**Comment by catborise** [comment]
yes it should be work without issue. But i open an issue for django-icons to update importlib-metadata requirements. i hope it will be fixed soon.
i tested on ubuntu before merge, it works as expected, it think, because ubuntu 20 provides python 3.10.

**Comment by choricc** [comment]
hello i met the same conflict .but i have slove it .
 my OS is Aonlis 8.6 
 python is 3.6
 i change the conf/requirments.txt
 
![QQ图片20221006130205](https://user-images.githubusercontent.com/51699030/194218305-da28f9a4-5eec-4577-9785-9e0b73862570.png)


> When I downloaded the latest version for installation, run "pip3 install -r conf/requirements.txt -q" , pip installation prompted a version conflict for importlib-metadata. log: ERROR: Cannot install -r conf/requirements.txt (line 22) and -r conf/requirements.txt (line 3) because these package versions have conflicting dependencies. The conflict is caused by: django-icons 22.1 depends on importlib-metadata<3; python_version < "3.8" markdown 3.4.1 depends on importlib-metadata>=4.4; python_version < "3.10"

**Comment by smallpize** [comment]
> hello i met the same conflict .but i have slove it . my OS is Aonlis 8.6 python is 3.6 i change the conf/requirments.txt
> 
> ![QQ图片20221006130205](https://user-images.githubusercontent.com/51699030/194218305-da28f9a4-5eec-4577-9785-9e0b73862570.png)
> 
> > When I downloaded the latest version for installation, run "pip3 install -r conf/requirements.txt -q" , pip installation prompted a version conflict for importlib-metadata. log: ERROR: Cannot install -r conf/requirements.txt (line 22) and -r conf/requirements.txt (line 3) because these package versions have conflicting dependencies. The conflict is caused by: django-icons 22.1 depends on importlib-metadata<3; python_version < "3.8" markdown 3.4.1 depends on importlib-metadata>=4.4; python_version < "3.10"

I delete the line for "markdown==3.3.1", run: 
> sed -i '/markdown==/d' conf/requirements.txt
> pip3 install -r conf/requirements.txt -q
> pip3 install markdown==3.4.1
